packages/aws-cdk-lib/awslint.json

```diff
@@ -885,6 +885,8 @@
     "props-physical-name:aws-cdk-lib.aws_apigateway.ResourceProps",
     "props-physical-name:aws-cdk-lib.aws_appsync.SourceApiAssociationProps",
     "docs-public-apis:aws-cdk-lib.aws_backup.Schedule",
-    "props-default-doc:aws-cdk-lib.aws_appsync.SourceApi.description"
+    "props-default-doc:aws-cdk-lib.aws_appsync.SourceApi.description",
+    "props-default-doc:aws-cdk-lib.aws_globalaccelerator.AcceleratorAttributes.ipAddresses",
+    "props-default-doc:aws-cdk-lib.aws_globalaccelerator.AcceleratorAttributes.ipAddressType"
```

Comment:
Is there any reason not to write default values in the interface without exclude by `awslint.json`? I would be willing to write it.